app/services/payment_providers/stripe_service.rb

```diff
@@ -28,43 +27,16 @@ def create_or_update(**args)
       stripe_provider.success_redirect_url = args[:success_redirect_url] if args.key?(:success_redirect_url)
       stripe_provider.save!
 
-      if secret_key != stripe_provider.secret_key
-        unregister_webhook(stripe_provider, secret_key)
-
-        PaymentProviders::Stripe::RegisterWebhookJob.perform_later(stripe_provider)
-      end
```

Comment:
Removed because graphql (the only place where this code is called) won't let you update the key. I think it's a relica from the past.

![CleanShot 2025-03-06 at 10 29 48@2x](https://github.com/user-attachments/assets/a9739a42-87dc-4547-9952-706d788aa6a2)